### ISSUE TITLE
the accuracy in the process of training is always zero

### ISSUE THREAD

**Issue by zhang-wen** [jadore801120/attention-is-all-you-need-pytorch#36]
i have problem here, when training, the accuracy on training data and validation data is always zeros, can anyone help me ? thanks a lot.

**Comment by fermat97** [comment]
Could you find the solution? I have the same problem. Thanks

**Comment by GeoffNN** [comment]
Same problem here, any fix?

**Comment by balagopal24** [comment]
Even I'm also facing the same problem..

**Comment by jadore801120** [comment]
Hello @zhang-wen , @fermat97 , @GeoffNN , @balagopal24, sorry for the error.
The code has been updated to PyTorch 0.4.1 and shall have resolved this problem.
Please take a look. If the problem goes on, feel free to reopen this issue.
Thanks,
Yu-Hsiang
